tfhe/benches/integer/signed_bench.rs

```diff
@@ -0,0 +1,764 @@
+    display_name: scalar_rotate_left,
+    rng_func: shift_scalar
+);
+
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: unchecked_scalar_mul_parallelized,
+    display_name: scalar_mul,
+    rng_func: default_scalar
+);
+
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: unchecked_scalar_bitand_parallelized,
+    display_name: scalar_bitand,
+    rng_func: default_scalar
+);
+
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: unchecked_scalar_bitor_parallelized,
+    display_name: scalar_bitor,
+    rng_func: default_scalar
+);
+
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: unchecked_scalar_bitxor_parallelized,
+    display_name: scalar_bitxor,
+    rng_func: default_scalar
+);
+
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: unchecked_scalar_eq_parallelized,
+    display_name: scalar_eq,
+    rng_func: default_scalar
+);
+
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: unchecked_scalar_ne_parallelized,
+    display_name: scalar_ne,
+    rng_func: default_scalar
+);
+
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: unchecked_scalar_le_parallelized,
+    display_name: scalar_le,
+    rng_func: default_scalar
+);
+
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: unchecked_scalar_lt_parallelized,
+    display_name: scalar_lt,
+    rng_func: default_scalar
+);
+
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: unchecked_scalar_ge_parallelized,
+    display_name: scalar_ge,
+    rng_func: default_scalar
+);
+
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: unchecked_scalar_gt_parallelized,
+    display_name: scalar_gt,
+    rng_func: default_scalar
+);
+
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: unchecked_signed_scalar_div_rem_parallelized,
+    display_name: scalar_div_rem,
+    rng_func: div_scalar
+);
+
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: unchecked_signed_scalar_div_parallelized,
+    display_name: scalar_div,
+    rng_func: div_scalar
+);
+
+criterion_group!(
+    unchecked_scalar_ops,
+    unchecked_scalar_left_shift_parallelized,
+    unchecked_scalar_right_shift_parallelized,
+    unchecked_scalar_rotate_right_parallelized,
+    unchecked_scalar_rotate_left_parallelized,
+    unchecked_scalar_bitand_parallelized,
+    unchecked_scalar_bitor_parallelized,
+    unchecked_scalar_bitxor_parallelized,
+    unchecked_scalar_mul_parallelized,
+    unchecked_scalar_eq_parallelized,
+    unchecked_scalar_ne_parallelized,
+    unchecked_scalar_le_parallelized,
+    unchecked_scalar_lt_parallelized,
+    unchecked_scalar_ge_parallelized,
+    unchecked_scalar_gt_parallelized,
+    unchecked_signed_scalar_div_rem_parallelized,
+    unchecked_signed_scalar_div_parallelized,
+    //unchecked_div_rem_floor_parallelized,
```

Comment:
too slow ?

Reply:
It's as slow as the other div_rem (a tiny bit slower to be exact), I commented it as I don't know if we want to bench it

Reply:
my guess is we do, but not sure if it does not break the time limit on our benchmarks

Reply:
forgot scalar variant here